pubspec.lock

```diff
@@ -328,7 +328,7 @@ packages:
       name: test_api
       url: "https://pub.dartlang.org"
     source: hosted
-    version: "0.2.15"
+    version: "0.2.11"
```

Comment:
I suspect this means you aren't on beta yet. You'll get errors when I land my change. See [here](https://github.com/flutter/flutter/wiki/Flutter-build-release-channels) for more detail. In short:

```
$ flutter channel beta
$ flutter upgrade
```

And then re-start your editor / IDE.

Reply:
Done! Looks like its working (and this lock file is updated)